source/UIAHandler/remote.py

```diff
@@ -4,36 +4,150 @@
 # Copyright (C) 2021-2022 NV Access Limited
 
 
-from typing import Optional, Union
-import os
-from ctypes import windll, byref, POINTER
-from comtypes.automation import VARIANT
-import NVDAHelper
+from typing import (
+	Optional,
+	Any,
+	Generator,
+	cast
+)
+from comtypes import GUID
 from comInterfaces import UIAutomationClient as UIA
+from ._remoteOps import remoteAlgorithms
+from ._remoteOps.remoteAPI import (
+	RemoteExtensionTarget,
+	RemoteInt
+)
+from ._remoteOps import operation
+from ._remoteOps import remoteAPI
+from ._remoteOps.lowLevel import (
+	TextUnit,
+	AttributeId,
+	StyleId
+)
 
 
 _dll = None
 
 
-def initialize(doRemote: bool, UIAClient: POINTER(UIA.IUIAutomation)):
+def initialize(doRemote: bool, UIAClient: UIA.IUIAutomation):
 	"""
 	Initializes UI Automation remote operations.
 	@param doRemote: true if code should be executed remotely, or false for locally.
 	@param UIAClient: the current instance of the UI Automation client library running in NVDA.
 	"""
-	global _dll
-	_dll = windll[os.path.join(NVDAHelper.versionedLibPath, "UIARemote.dll")]
-	_dll.initialize(doRemote, UIAClient)
+	return True
 
 
 def msWord_getCustomAttributeValue(
-		docElement: POINTER(UIA.IUIAutomationElement),
-		textRange: POINTER(UIA.IUIAutomationTextRange),
-		customAttribID: int
-) -> Optional[Union[int, str]]:
-	if _dll is None:
-		raise RuntimeError("UIARemote not initialized")
-	customAttribValue = VARIANT()
-	if _dll.msWord_getCustomAttributeValue(docElement, textRange, customAttribID, byref(customAttribValue)):
-		return customAttribValue.value
-	return None
+	docElement: UIA.IUIAutomationElement,
+	textRange: UIA.IUIAutomationTextRange,
+	customAttribID: int
+) -> Optional[Any]:
+	guid_msWord_extendedTextRangePattern = GUID("{93514122-FF04-4B2C-A4AD-4AB04587C129}")

```

Comment:
Likewise these could also should be constants in the final version.

Reply:
I would argue these are already constants. They are not used anywhere else, and already have descriptive names.